tools/test-v8.bat

```diff
@@ -0,0 +1,45 @@
+if not defined DEPOT_TOOLS_PATH goto depot-tools-not-found
+if "%config%"=="Debug" set test_args=%target_arch%.debug
+if "%config%"=="Release" set test_args=%target_arch%.release
+set savedpath=%path%
+set path=%DEPOT_TOOLS_PATH%;%path%
```

Comment:
Now you can `SETLOCAL` and `ENDLOCAL` in `:test-v8-exit` and then ___I___ don't care what monstrosities you do to poor old `Path`